### PR TITLE
Feat/63 code of conduct decision

### PR DESCRIPTION
**Pull request by Monstarrrr**

### CODE REVIEW

File: docs/decisions/governance/code-of-conduct_DECISION.md [hunk 1]
```diff
@@ -8,29 +8,54 @@
...
+5. No steps provided to report violations
+6. Consequences are defined
+   - Correction
+   - Warning
+   - Temporary ban
+   - Permanent ban
+7. Inclusivity emphasized multiple times throuhought the document
+8. Not community input options provided
+
+**Reason(s)**: A basic code of conduct (heavily inspired by the default github code of conduct) should suffice especially since we don't have a community yet anyways.
```
Comment (purple-void) [inline docs/decisions/governance/code-of-conduct_DECISION.md:52]: maybe remove this? ["should" ](https://www.psychologytoday.com/us/blog/the-path-passionate-happiness/202205/why-no-good-comes-the-word-should)

File: docs/decisions/governance/code-of-conduct_DECISION.md [hunk 2]
```diff
@@ -8,29 +8,54 @@
...
+   - Attribution
+4. Tone is neutral not positive
+5. No steps provided to report violations
+6. Consequences are defined
+   - Correction
+   - Warning
+   - Temporary ban
+   - Permanent ban
+7. Inclusivity emphasized multiple times throuhought the document
+8. Not community input options provided
```
Comment (purple-void) [inline docs/decisions/governance/code-of-conduct_DECISION.md:50]: do you mean "No community input options provided"?

File: docs/decisions/governance/code-of-conduct_DECISION.md (hunk 1 above)
Comment (Monstarrrr) [reply docs/decisions/governance/code-of-conduct_DECISION.md:52]: That's an interesting point in general, although there exists specifications for the use of such word in formal documents, which I believe is a different context than the link you provided.  

I would propose to follow the meanings as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt):  

> SHOULD: This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.  

Basically in this example, a basic code of conduct does suffice in the current circumstances, but that could change if the circumstances were to change (e.g. new contributors to the project joined).

Comment (purple-void) [reply docs/decisions/governance/code-of-conduct_DECISION.md:52]: That's interesting. I've never seen that definition. I don't agree with anything that suggests we "have to" do anything https://youtu.be/jyqGGAn2R1E?feature=shared, such as "must" in your should definition and "should" in the way I was perceiving it earlier.

Everything else looks fine.

**Comment by Monstarrrr** [comment]
@purple-void Once you're done reviewing please add your confidence level and i'll merge

File: docs/governance/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ We pledge to act and interact in ways that contribute to an **open**, **welcomin
 
 ## Our Standards
 
-Examples of **required** behavior that contributes to a positive environment for our
+Examples of **standard** behavior that contributes to a positive environment for our
```
Comment (purple-void) [inline docs/governance/CODE_OF_CONDUCT.md:17]: "required" is more clear to us. we equate "standard" to "common" and don't always follow what's common.

File: docs/decisions/governance/contributing_DECISION.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Lack of clear guidance affects the growth of the project's community and may result in **missed opportunities** for **valuable contributions**.
+
+There is a need to provide clear and concise **guidance** to potential contributors, **outlining** the various ways they can contribute and **the steps** they need to follow to **ensure their contributions are successful**.
+
+## Decision Drivers <!-- optional -->
+
+- **Clarity**: The contributing.md file should provide clear and concise instructions on how individuals can contribute to the project. Clear communication ensures that contributors understand the process and can engage effectively.
+- **Accessibility**: It's important to make the contributing.md file easily accessible to potential contributors. This involves placing it prominently in the project repository and ensuring it's easy to understand for individuals with varying levels of technical expertise.
+- **Inclusivity**: The contributing.md file should foster an inclusive environment by welcoming contributions from individuals of diverse backgrounds and skill levels. It should avoid using exclusive language or assumptions about contributors' knowledge.
```
Comment (purple-void) [inline docs/decisions/governance/contributing_DECISION.md:24]: maybe add line breaks between the first three for visual consistency

File: docs/decisions/governance/contributing_DECISION.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - [@Monstarrrr](https://github.com/Monstarrrr)
+- **Technical Story**: [related URLs] e.g.
+  - Pull Request #58 - ["Write CODE_OF_CONDUCT.md"](https://github.com/Monstarrrr/rebutify/pull/58) <!-- optional -->
+- **Last updated**: [DD-MM-YYYY]
+
+## Context and Problem Statement
+
+Lack of clear guidance affects the growth of the project's community and may result in **missed opportunities** for **valuable contributions**.
+
+There is a need to provide clear and concise **guidance** to potential contributors, **outlining** the various ways they can contribute and **the steps** they need to follow to **ensure their contributions are successful**.
```
Comment (purple-void) [inline docs/decisions/governance/contributing_DECISION.md:18]: we don't "need" to do anything

we rephrased the lines:

> Lack of clear guidance affects the growth of the project's community and may result in **missed opportunities** for **valuable contributions**.
> 
> We want to provide clear and concise **guidance** to potential contributors, **outlining** the various ways they can contribute and **the steps**  they can follow to **ensure their contributions are successful**.

File: docs/governance/CODE_OF_CONDUCT.md (hunk 3 above)
Comment (Monstarrrr) [reply docs/governance/CODE_OF_CONDUCT.md:17]: I agree, it felt odd when I read it in the moment that's why I replaced it but I can see it making more sense.

**Review by purple-void — APPROVED** [review]
Okay. implicit talking
